election/settings.py

```diff
@@ -29,6 +35,7 @@
     'voterecode',
```

Comment:
corsheaders is not in the APP

Reply:
concerns raised 
1 -" This module "corsheaders" is in the project app configuration but there is no such app."
The initial approach for communication between the two projects was with Ajax,a cross origin scripting issue arose hence , I had to let utilize a python module called CORs ,unfortunately flask and django communication over ajax introduced new issues.
Will clean cors module in my next PR.